ui/components/multichain/account-list-menu/account-list-menu.js

```diff
@@ -64,6 +66,48 @@ const ACTION_MODES = {
   IMPORT: 'import',
 };
 
+function getLabel(type, account) {
+  switch (type) {
+    case KeyringType.qr:
+      return HardwareKeyringNames.qr;
+    case KeyringType.imported:
+      return translate('imported');
+    case KeyringType.trezor:
+      return HardwareKeyringNames.trezor;
+    case KeyringType.ledger:
+      return HardwareKeyringNames.ledger;
+    case KeyringType.lattice:
+      return HardwareKeyringNames.lattice;
+    ///: BEGIN:ONLY_INCLUDE_IF(keyring-snaps)
+    case KeyringType.snap:
+      if (account.metadata.snap?.name) {
+        return `${account.metadata.snap?.name} (${translate('beta')})`;
+      }
+      return `${translate('snaps')} (${translate('beta')})`;
+    ///: END:ONLY_INCLUDE_IF
+    default:
+      return null;
+  }
+}
```

Comment:
I think you should move this method to an utils file instead of having it in a component and then just import it in the appropriate views/components

Reply:
Good Idea, done!